spec/features/admin/councils/new_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Council', type: :feature, js: true do
+  let!(:council) { FactoryBot.create :council }
+
+  context 'anonymous' do
+    it "can't see it" do
+      visit "/admin/councils/#{council.id}"
+      expect(page).to have_text('Rates Rebate 2018/2019')
+      expect(page).to have_text('Log in')
+    end
+  end
+
+  context 'can edit' do
```

Comment:
The context is 'can edit' but this test doesn't actually show that a council can be edited. Can you update it accordingly? (Capybara lets you fill_in fields, press Submit and then test the response.)

Reply:
Oh wait, I see, this is the new spec ... Maybe the context should be 'edit form is displayed' to make it clearer?